src/Backend/Modules/Pages/Installer/Installer.php

```diff
@@ -356,28 +181,123 @@ private function installExampleData(): void
                     [
                         'html' => __DIR__ . '/Data/' . $language . '/lorem_ipsum.txt',
                     ],
-                    ['extra_id' => $extras['search_form'], 'position' => 'top']
+                    ['extra_id' => $this->getExtraId('sitemap'), 'position' => 'top']
                 );
             }
         }
     }
 
-    private function setRights(): void
+    private function configureFrontendExtras(): void
     {
-        // module rights
-        $this->setModuleRights(1, $this->getModule());
+        $this->extraIds['search'] = $this->insertExtra('Search', ModuleExtraType::block(), 'Search');
```

Comment:
The problem here is that this also get added in the search installer, what results in having this widget more than once in the modules_extras table.